t/pod_coverage.t

```diff
@@ -11,4 +11,7 @@ my @await = (
   qw(AWAIT_NEW_FAIL AWAIT_ON_CANCEL AWAIT_ON_READY AWAIT_WAIT)
 );
 
-all_pod_coverage_ok({also_private => ['BUILD_DYNAMIC', @await, 'spurt']});
+# base utils only to be used in Mojo::Base and should not be used elsewhere
```

Comment:
Nitpick: Comment capitalisation.

Reply:
Is that correct english? Sounds a little wrong to me.

Reply:
Reworded to 
> These are base utils only to be used in Mojo::Base and not elsewhere